frontend/src/pages/AnnotateTemplate.tsx

```diff
@@ -94,24 +99,24 @@ const AnnotateTemplate: React.FC = () => {
     };
 
     convertPdfToImages(pdfFile).then((imgs) => {
+      // TODO: This is redundant, we should just store the images in the store
       setImages(imgs);
-      localStorage.setItem('images', JSON.stringify(imgs));
+      storeImages(imgs);
     });
   }, [files, pdfFile]);
   useEffect(() => {
     const getImage = async () => {
-      const localImages = await JSON.parse(
-        localStorage.getItem('images') || "[]"
-      );
+      const localImages = baseImages;
       if (localImages && localImages.length > 0) {
-        setImages(localImages.images);
+        setImages(localImages);
       }
     };
 
     const handleUnmount = () => {
       setDrawnFields(new Set());
       setSelectedField(null);
     }
+    // TODO: Ask Kevin what this is for, it's not clear
```

Comment:
oh after we annotate i noticed that we need to clear those fields to prepare the the next annotation